src/test/kotlin/net/atos/zac/app/planitems/PlanItemsRESTServiceTest.kt

```diff
@@ -275,6 +276,75 @@ class PlanItemsRESTServiceTest : BehaviorSpec({
 
+    Given("Additional info human task with a fatal date after the fatal date of the related zaak") {
+        val numberOfDays = 3L
+        val additionalInfoPlanItemInstanceId = FormulierDefinitie.AANVULLENDE_INFORMATIE.toString()
+        val restHumanTaskData = createRESTHumanTaskData(
+            planItemInstanceId = additionalInfoPlanItemInstanceId,
+            taakdata = mapOf(
+                "dummyKey" to "dummyValue"
+            ),
+            fataledatum = LocalDate.now().plusDays(numberOfDays)
+        )
+        val zaak = createZaak(
+            zaakTypeURI = URI("http://example.com/$zaakTypeUUID"),
+            uiterlijkeEinddatumAfdoening = LocalDate.now()
+        )
+        val extendedZaak = createZaak(
+            zaakTypeURI = URI("http://example.com/$zaakTypeUUID"),
+            uiterlijkeEinddatumAfdoening = LocalDate.now().plusDays(numberOfDays)
+        )
+        val zaakafhandelParametersMock = mockk<ZaakafhandelParameters>()
+
+        every { cmmnService.readOpenPlanItem(additionalInfoPlanItemInstanceId) } returns planItemInstance
+        every { zaakVariabelenService.readZaakUUID(planItemInstance) } returns zaak.uuid
+        every { zrcClientService.readZaak(zaak.uuid) } returns zaak
+        every { policyService.readZaakRechten(zaak) } returns createZaakRechtenAllDeny(startenTaak = true)
+        every {
+            zaakafhandelParameterService.readZaakafhandelParameters(zaakTypeUUID)
+        } returns zaakafhandelParametersMock
+        every { planItemInstance.planItemDefinitionId } returns additionalInfoPlanItemInstanceId
+        every {
+            zaakafhandelParametersMock.findHumanTaskParameter(additionalInfoPlanItemInstanceId)
+        } returns Optional.of(
+            createHumanTaskParameters().apply {
+                doorlooptijd = 10
+            }
+        )
+        every {
+            suspensionZaakHelper.extendZaakFatalDate(zaak, numberOfDays, "Aanvullende informatie opgevraagd")
+        } returns extendedZaak
+        every {
+            cmmnService.startHumanTaskPlanItem(
+                additionalInfoPlanItemInstanceId,
+                restHumanTaskData.groep.id,
+                null,
+                DateTimeConverterUtil.convertToDate(restHumanTaskData.fataledatum),
+                restHumanTaskData.toelichting,
+                any(),
+                zaak.uuid
+            )
+        } just runs
+        every { indexingService.addOrUpdateZaak(zaak.uuid, false) } just runs
+
+        When("A human task plan item is started") {
+            planItemsRESTService.doHumanTaskplanItem(restHumanTaskData)
+
+            Then("The task is created with its own fatal date") {
+                verify(exactly = 1) {
+                    cmmnService.startHumanTaskPlanItem(any(), any(), any(), any(), any(), any(), any())
+                    indexingService.addOrUpdateZaak(any(), any())
+                }
+            }
+
+            And("zaak extend fatal date was performed") {
```

Comment:
Ah, so now the zaak's fatal date should be extended and should be the same as the task's fatal date? I don't really see this back in this test code? We only verify if the correct method is called but not if the actual fatal date was set correctly? Maybe that's still possible to add using a `slot` for the method argument(s)?

Reply:
We do have tests for that in the `SuspensionZaakHelperTest`, so decided to check only if this logic was called at all here